File: ProjectManagementToolReact/appPages/Issue.js
import React from 'react';

export default class Issue {
    name: string;
    sprint: string;

    constructor(name:string,sprint:string) {
        this.name = name;
        this.sprint = sprint;
    }

    getName(){
        return this.name;
    }

    getSprint(){
        return this.sprint;
    }

    setName(name:string){
        this.name = name;
    }

    setSprint(sprint:string){
        this.sprint = sprint;
    }
}

File: ProjectManagementToolAndroidStudio/app/src/main/java/core/projectmanagementtool/model/Issue.java
package core.projectmanagementtool.model;

import java.io.Serializable;
import org.json.JSONObject;
import org.json.JSONException;

/**
 * Created by tudor on 04-Nov-17.
 */

public class Issue implements Serializable {
    private String name;
    private String sprint;

    public Issue(){
        this.name = "NoArgsConst";
        this.sprint = "yup";
    }

    public Issue(String name,String sprint){
        this.name = name;
        this.sprint = sprint;
    }

    public String getName(){
        return this.name;
    }

    public String getSprint(){
        return this.sprint;
    }

    public void setName(String name){
        this.name = name;
    }

    public void setSprint(String sprint){
        this.sprint = sprint;
    }

    public String toJSON() {
        JSONObject jsonObject = new JSONObject();
        try {
            jsonObject.put("name", getName());
            jsonObject.put("sprint", getSprint());

            return jsonObject.toString();
        } catch (JSONException e) {
            // TODO Auto-generated catch block
            e.printStackTrace();
            return "";
        }
    }

    @Override
    public String toString(){
        return this.name;
    }

}
